pkg/service/types.go

```diff
@@ -16,6 +16,9 @@ type MeshService struct {
 
 	// The name of the service
 	Name string
+
+	// The cluster domain in which the service runs
+	ClusterDomain string
 }
 
 func (ms MeshService) String() string {
```

Comment:
We'll also need this function to take into consideration the cluster name.

And we'll want to create a new one that returns this value, ie: UnscopedCluster()

Reply:
check out https://github.com/steeling/osm/commit/b0dede3453d857439a7e3a8739b2d455037ea184 for `UnscopedName` in every place we want to use that